openapi.json

```diff
@@ -1685,66 +1685,66 @@
-                    "versions": {
-                      "3.1.0": {
-                        "libraries": {
-                          "Rcpp": {
-                            "version": "1.0.10",
-                            "links": [
-                              {
-                                "href": "https://cran.r-project.org/web/packages/Rcpp/index.html"
-                              }
-                            ]
-                          },
-                          "rmarkdown": {
-                            "version": 1.7,
-                            "links": [
-                              {
-                                "href": "https://cran.r-project.org/web/packages/rmarkdown/index.html"
-                              }
-                            ]
-                          }
+                  "example": {
+                    "PHP7": {
+                      "description": "Just an example how to reference a docker image.",
+                      "docker": "openeo/udf-php7",
+                      "default": "latest",
+                      "tags": [
+                        "latest",
+                        "7.3.1",
+                        "7.3",
+                        "7.2"
+                      ],
+                      "links": [
+                        {
+                          "href": "https://hub.docker.com/openeo/udf-php7/"
                         }
-                      },
-                      "3.5.2": {
-                        "libraries": {
-                          "Rcpp": {
-                            "version": "1.2.0",
-                            "links": [
-                              {
-                                "href": "https://cran.r-project.org/web/packages/Rcpp/index.html"
-                              }
-                            ]
-                          },
-                          "rmarkdown": {
-                            "version": 1.7,
-                            "links": [
-                              {
-                                "href": "https://cran.r-project.org/web/packages/rmarkdown/index.html"
-                              }
-                            ]
+                      ]
+                    },
+                    "R": {
+                      "description": "R programming language with Rcpp and rmarkdown.",
+                      "default": "3.5.2",
+                      "versions": {
+                        "3.1.0": {
+                          "libraries": {
+                            "Rcpp": {
+                              "version": "1.0.10",
+                              "links": [
+                                {
+                                  "href": "https://cran.r-project.org/web/packages/Rcpp/index.html"
+                                }
+                              ]
+                            },
+                            "rmarkdown": {
+                              "version": "1.7",
```

Comment:
I guess this was wrong due to the fact that I usually edit the files in YAML and then convert them to JSON. This conversion probably leads to a number instead of a string. So to ensure it stays a string we should change it to "1.7.0". Could you make this change?

Reply:
ok will do